fittable.R

```diff
@@ -0,0 +1,36 @@
+fittable <- function(..., indices="robust", all=F) {
+  require(lavaan)
+  models <- list(...)
+  indic.names <- c("x2", "df", "p", "BIC", "CFI", "TLI", "RMSEA", "lower.RMSEA", "upper.RMSEA", "SRMR")
+  
+  if (indices == "scaled") {
+    indic <- c("chisq.scaled", "df.scaled", "pvalue.scaled", "bic", "cfi.scaled", "tli.scaled", "rmsea.scaled",
+               "rmsea.ci.lower.scaled", "rmsea.ci.upper.scaled", "srmr")
+
+  } else if (indices == "robust") {
+    indic <- c("chisq.scaled", "df.scaled", "pvalue.scaled", "bic", "cfi.robust", "tli.robust", "rmsea.robust",
+               "rmsea.ci.lower.robust", "rmsea.ci.upper.robust", "srmr")
+  } else {
+    indic.names <- indic <- indices
+  }
+
+  table.fit <- NULL
+  all.fit <- list()
+  for (m in c(1:length(models))) {
+    all.fit[[m]] <- fitMeasures(models[[m]])

```

Comment:
btw doporucuju implicitne odkazovat na konkretni package, vzdycky za vsech pripadu. `lavaan::fitMeasures(models[[m]])`

Ma to dva primarni ucely:

1) Vzdycky zajistis spravny scoping, i kdybys mel nactenou nejakou jinou package ktera obsahuje stejnou funkci, coz se klidne muze stat za rok, dva, tri, nikdy nevis
2) Pomaha to pri cteni kodu nekym jinym, kazdy pak vi, co za funkci jsi vlastne chtel pouzit

Reply:
Jo, to je dobrý postřeh!